tamagotchi/__main__.py

```diff
@@ -13,11 +13,11 @@
 
 import tqdm
 import appdirs
-from telethon import TelegramClient, utils
+from garry import TelegramClient, utils
```

Comment:
You even took into consideration the fact that you renamed `telethon` → `garry`… Nice touch.